09-mapping.Rmd

```diff
@@ -14,10 +14,18 @@ library(spData)
 library(spDataLarge)
 ```
 
-- In addition, it uses the following visualization packages (also install shiny if you want to develop interactive mapping applications):
+- The main package used in this chapter is **tmap**. 
+We recommend you to install its development version from the [r-universe](https://r-universe.dev/) repository, which is updated more frequently than the CRAN version:
+
+```{r}
+# eval: false
+install.packages("tmap", repos = c("https://r-tmap.r-universe.dev",
```

Comment:
Would be good to use the CRAN version when tmap 4 released.

Reply:
True, but it probably won't happen before the end of the year...